app/src/main/java/org/greenstand/android/TreeTracker/signup/CredentialEntryView.kt

```diff
@@ -105,11 +115,11 @@ fun CredentialEntryView(viewModel: SignupViewModel, state: SignUpState) {
                     placeholder = { Text(text = stringResource(id = R.string.email_placeholder), color = Color.White) },
                     keyboardOptions = KeyboardOptions(
                         keyboardType = KeyboardType.Email,
-                        imeAction = ImeAction.Go,
+                        imeAction = ImeAction.Done,
```

Comment:
Why did you change this? 'done' usually means you completed something, but in this case we go to the next input.